test/suites/storage-version-migration.robot

```diff
@@ -0,0 +1,92 @@
+*** Settings ***
+Documentation       Storage version migration test suite.
+
+Library             Process
+Library             ../resources/DataFormats.py
+Resource            ../resources/oc.resource
+Resource            ../resources/common.resource
+Resource            ../resources/kubeconfig.resource
+Resource            ../resources/microshift-process.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${BETA_CRD}         assets/storage-version-migration/crd.beta.yaml
+${STABLE_CRD}       assets/storage-version-migration/crd.stable.yaml
+${CR_RESOURCE}      assets/storage-version-migration/cr.yaml
+
+
+*** Test Cases ***
+Storage Version Migration Test
+    [Documentation]    Verify that storage migrations get created when CRDs get updated.
+    [Tags]    smoke
```

Comment:
```suggestion
    [Tags]    smoke    restart    slow
```

Nit: We should think about applying some consistent tagging to all of the tests. There's no reason to hold this up for that, though.

Reply:
Ah, thats a good point, I forgot to ask about the tags. Since @pmtk will look at this tomorrow, let me add those in there now.